spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,29 @@
+feature "View Peep" do
+  scenario "visiting the index page" do
+    visit("/")
+    expect(page).to have_content "Chitwits"
+  end
+
+  scenario "a user can see all the Peeps" do
+    visit("/chitcreek")
+    click_button("Peep a chit")
+    fill_in("new_chit", with: "This is a Peep")
+    click_button("Peep!")
+    expect(page).to have_content "This is a Peep"
+  end
+
+  scenario "a user sees Peeps in reverse order" do
+    visit("/chitcreek")
+    click_button("Peep a chit")
+    fill_in("new_chit", with: "This is a Peep")
+    click_button("Peep!")
+  click_button("Peep a chit")
+  fill_in("new_chit", with: "This is another Peep")
+  click_button("Peep!")
+  click_button("Peep a chit")
+  fill_in("new_chit", with: "This is the lucky third Peep")
+  click_button("Peep!")
+
+    expect(page).to have_content "This is the lucky third Peep, This is another Peep, This is a Peep"
```

Comment:
I guess you want to test for these three existing in the page in this order, rather than tying your front-end to the use of commas. Perhaps you could use html id tags which increase in steps of 1, and then test the content of each of those. Or you could use `.to_s.split` on the page's html content.